source/MaterialXGenShader/TypeDesc.h

```diff
@@ -13,6 +13,8 @@
 #include <MaterialXCore/Value.h>
 #include <MaterialXCore/Document.h>
 
+#include <cstdint>
+#include <stdint.h>
```

Comment:
@luyatshimbalanga I can see the need for `cstdint` in new versions of GCC, but what is the motivation for additionally including `stdint.h` in just this one source file?

Reply:
Thar was a suggestion from the compiler when it encounter the failure on that source. If `sdtint.h` is redundant, feel free to remove it.

Reply:
Thanks, @luyatshimbalanga, and I can make that change over here.  Structurally, we may also want to add a single include of `cstdint` in Library.h within MaterialXCore, rather than including it in each source file that references a fixed-width type.

Reply:
I've gone ahead and made that simplification to the current PR, @luyatshimbalanga, and let me know if this continues to build successfully in your GCC 15 environment.

Reply:
The simplified PR modification runs successfully in the GCC build environment as seen on 
https://download.copr.fedorainfracloud.org/results/@designsuite/blender/fedora-rawhide-x86_64/08882889-materialx/builder-live.log